deps/wazuh_testing/wazuh_testing/tools/agent_simulator.py

```diff
@@ -1036,70 +1302,63 @@ def keep_alive(self):
                   ((time() - start_time) %
                    self.agent.modules["keepalive"]["frequency"]))
 
-    def fim(self):
-        """Send a File Integrity Monitoring message from the agent to the manager."""
-        sleep(10)
-        start_time = time()
-        # Loop events
-        while self.stop_thread == 0:
-            event = self.agent.create_event(self.agent.fim.get_message())
-            self.sender.send_event(event)
-            self.totalMessages += 1
-            if self.totalMessages % self.agent.modules["fim"]["eps"] == 0:
-                sleep(1.0 - ((time() - start_time) % 1.0))
-
-    def fim_integrity(self):
-        """Send an integrity FIM message from the agent to the manager"""
+    def run_module(self, module):
+        """Send a module message from the agent to the manager.
+         Args:
```

Comment:
```suggestion

         Args:
```

Reply:
This is unresolved yet.

Reply:
It was resolved by @Rebits a few days ago: https://github.com/wazuh/wazuh-qa/commit/6976fb26ae1e4fcd0d905a309eb1820db1111a5c